internal/dag/httpproxy_processor.go

```diff
@@ -526,6 +481,8 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 	}
 	insecure.RateLimitPolicy = rlp
 
+	insecure.GlobalExternalAuthorization = p.computeVirtualHostAuthorization(validCond, proxy)
```

Comment:
I think we still want to make a call to `p.computeVirtualHostAuthorization` here, to validate the refs and set conditions on the HTTPProxy if needed, but per previous comment, we don't actually need to save the config in the DAG for non-TLS vhosts.